Sources/Target/FreeBSD/Process.cpp

```diff
@@ -201,8 +201,7 @@ ErrorCode Process::wait() {
 
   switch (_currentThread->_stopInfo.event) {
   case StopInfo::kEventNone:
-    switch (_currentThread->_stopInfo.reason) {
-    case StopInfo::kReasonNone:
+    if (_currentThread->_stopInfo.reason == StopInfo::kReasonNone) {
```

Comment:
Ahah :D I don't know if it make sense. But I can't really test, and we don't have tests, so I try to fix compilation with changing any behavior. If it's ok I will prefer to keep it like that until someone review/clean the freebsd code.